example/weather/services/tester/run_tests.go

```diff
@@ -77,6 +77,24 @@ func recoverFromTestPanic(ctx context.Context, testName string, testCollection *
 	}
 }
 
+// Filters a testMap based on a test name that is a glob string
+// using standard wildcards https://tldp.org/LDP/GNU-Linux-Tools-Summary/html/x11655.htm
+func matchTestFilter(ctx context.Context, test string, testMap map[string]func(context.Context, *TestCollection)) (bool, []func(context.Context, *TestCollection)) {
+	match := false
+	testMatches := []func(context.Context, *TestCollection){}
```

Comment:
There's no need to initialize slices like this in Go, it's OK to append to `nil` so this could just be:
```go
var testMatches []func(context.Context, *TestCollection)
```

Reply:
👍🏻